generators/newapp/templates/dot-dockerignore.tmpl

```diff
@@ -0,0 +1,3 @@
+node_modules/
+*.log
+bin/
```

Comment:
Another thing we could consider is using a [multi-stage Dockerfile](https://docs.docker.com/engine/userguide/eng-image/multistage-build/#before-multi-stage-builds) that will end up with a slimmer docker image, but i think is ok to leave it for the future.